docs/queue/kafka.md

```diff
@@ -0,0 +1,63 @@
+# Managing Kafka queue configuration
+
+## Create a Kafka queue configuration
+
+Let's start by defining a single queue with a simple consumer and producer:
+
+```java
+import io.orkes.conductor.client.model.event.QueueConfiguration;
+import io.orkes.conductor.client.model.event.QueueWorkerConfiguration;
+import io.orkes.conductor.client.model.event.kafka.KafkaConfiguration;
+import io.orkes.conductor.client.model.event.kafka.KafkaConsumer;
+import io.orkes.conductor.client.model.event.kafka.KafkaProducer;
+
+private static final String KAFKA_QUEUE_TOPIC_NAME = "test_sdk_java_kafka_queue_name";
```

Comment:
Let's also ask users to create this topic in their Kafka cluster before configuring/trying to work with a topic-based queue